compiler-rt/lib/builtins/aarch64/sme-libc-routines.c

```diff
@@ -22,3 +23,46 @@ const void *__arm_sc_memchr(const void *src, int c,
 
   return NULL;
 }
+
+#ifndef __ARM_FEATURE_UNALIGNED
+
+static void *memcpy_fwd(void *dest, const void *src,
```

Comment:
Is it still worth adding the `__arm_sc_` prefix to these functions to keep all functions in the same namespace? Just worried about potential clashes with other target implementations.

Reply:
OK yeah I see. If it's ok with you, I do think it's better to have the same naming scheme for all the functions and start with `__arm_sc_`

Reply:
Done. Thanks